### ISSUE TITLE
'eh-hover' class not implemented

### ISSUE THREAD

**Issue by umdstu** [cytoscape/cytoscape.js-edgehandles#140]
I've actually tried to implement this a couple of different ways myself as I wanted this behavior, but could never get it to work aside from setting the documents cursor style dynamically on mouseover. While working on the PR for #138, I noticed the eh-hover class documented in the README, but the class is never added to the handle, only removed in the stop() function. I don't even see where the hover event is emit'ed for the handle.

TLDR; .eh-hover doesn't actually seem implemented.
